Library/Homebrew/dev-cmd/contributions.rb

```diff
@@ -17,14 +17,14 @@ module Homebrew
   sig { returns(CLI::Parser) }
   def contributions_args
     Homebrew::CLI::Parser.new do
-      usage_banner "`contributions` <email|name> [<--repositories>`=`]"
+      usage_banner "`contributions` <email|name> [<--repositories|--repos>`=`]"
```

Comment:
```suggestion
      usage_banner "`contributions` <email|name> [<--repositories>`=`]"
```

I don't think we tend to document all possible combinations here.